test/dynamo/test_optimizers.py

```diff
@@ -99,6 +99,21 @@ class OptimizerTests(torch._dynamo.test_case.TestCase):
     setattr(OptimizerTests, "test_" + opt.__name__.lower(), make_test(opt))
 
 
+class MyOptimizer(torch.optim.Optimizer):
+    def __init__(self, params, val):
+        super().__init__(params, {})
+        self.val = val
+
+    def _init_group(self):
+        return self.val
```

Comment:
This is actually not reflective of the current use of `_init_group`. In the current use, the returned values are always derived from guarded tensor metas.

Reply:
We can have more general tests for @init_values_once once it is implemented

Reply:
done